src/query/query_parser/query_parser.rs

```diff
@@ -401,7 +405,7 @@ impl QueryParser {
                 Ok(Term::from_field_bytes(field, &bytes))
             }
             FieldType::IpAddr(_) => Err(QueryParserError::UnsupportedQuery(
-                "Range query are not supported on IpAddr field.".to_string(),
+                "Range query are not supported on ip field.".to_string(),
```

Comment:
Is there an issue open for this?

Reply:
https://github.com/quickwit-oss/tantivy/issues/1626